src/queue.c

```diff
@@ -32,6 +32,7 @@
 #endif
 #if HAVE_PTHREAD_WORKQUEUES && (!HAVE_PTHREAD_WORKQUEUE_QOS || DISPATCH_DEBUG) && \
 		!HAVE_PTHREAD_WORKQUEUE_SETDISPATCH_NP && \
+		!HAVE_INTERNAL_PTHREAD_WORKQUEUE && \
```

Comment:
I'm not very clear on why HAVE_PTHREAD_WORKQUEUES is true for the internal workqueue case, that doesn't seem quite right, that define is intended to guard the use of the specific pthread_workqueue API being used (which the internal version doesn't emulate). It would be easier to understand if the cases where you need something that is guarded by HAVE_PTHREAD_WORKQUEUES just added a || HAVE_INTERNAL_PTHREAD_WORKQUEUE instead (and then you wouldn't need the confusing addition here)